app-frontend/src/app/pages/projects/edit/annotate/import/import.controller.js

```diff
@@ -0,0 +1,96 @@
+/* globals _, $ */
+
+export default class AnnotateImportController {
+    constructor( // eslint-disable-line max-params
+        $log, $state, $scope, $timeout
+    ) {
+        'ngInject';
+        this.$log = $log;
+        this.$state = $state;
+        this.$scope = $scope;
+        this.$timeout = $timeout;
+        this.$parent = $scope.$parent.$ctrl;
+    }
+
+    $onInit() {
+        this.matchKeys = {};
+        this.bindUploadEvent();
+        this.isMachineData = false;
+        this.$scope.$on('$destroy', this.$onDestroy.bind(this));
+    }
+
+    $onDestroy() {
+        this.matchKeys = {};
+        this.isMachineData = false;
+    }
+
+    /* eslint-disable no-undef */
+    bindUploadEvent() {
+        $('#btn-upload').change((e) => {
+            let upload = _.values(e.target.files);
+            if (upload.length) {
+                _.forEach(upload, (datum) => {
```

Comment:
I believe you could use the es6 forEach here rather than the lodash one: `upload.forEach(datum =>`